minorminer/utils/parallel_embeddings.py

```diff
@@ -101,17 +103,46 @@ def embeddings_to_array(embs: list, node_order=None, as_ndarray=False):
         return [[emb[v] for v in node_order] for emb in embs]
 
 
+def array_to_embeddings(embs: list, node_order=None):
+    """Convert list of embedding lists (values) to dictionary
+
+    Args:
+        embs: A list of embeddings, each embedding is a list where values are
+            chains in node order.
+        node_order: An iterable giving the ordering of
+            variables in each row. When not provided variables are ordered to
+            match the first embedding :code:``embs[0].keys()``. node_order
+            can define any subset of the source graph nodes (embedding
+            keys).
+
+    Returns:
+        An embedding dictionary
+    """
+    if len(embs) is None:
+        return []
```

Comment:
`len()` never evaluates to `None`.
```suggestion
    if not embs:
        return []
```